public/apps/configuration/utils/test/password-edit-panel.test.tsx

```diff
@@ -24,18 +39,47 @@ describe('Password edit panel', () => {
   const updateIsInvalid = jest.fn();
   const useState = jest.spyOn(React, 'useState');
   const useEffect = jest.spyOn(React, 'useEffect');
+  const mockCoreStart = {
+    http: 1,
+  };
 
   beforeEach(() => {
     useEffect.mockImplementationOnce((f) => f());
     useState.mockImplementation((initialValue) => [initialValue, setState]);
+    (getDashboardsInfo as jest.Mock).mockImplementation(() => {
+      return mockDashboardsInfo;
+    });
     component = shallow(
-      <PasswordEditPanel updatePassword={updatePassword} updateIsInvalid={updateIsInvalid} />
+      <PasswordEditPanel
+        coreStart={mockCoreStart as any}
+        updatePassword={updatePassword}
+        updateIsInvalid={updateIsInvalid}
+      />
     );
   });
 
-  it('renders', () => {
-    expect(updatePassword).toHaveBeenCalledTimes(1);
-    expect(updateIsInvalid).toHaveBeenCalledTimes(1);
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders', (done) => {
+    (getDashboardsInfo as jest.Mock).mockImplementation(() => {
+      return mockDashboardsInfo;
+    });
```

Comment:
Is this mock needed?

Reply:
Looks like this was made redundant by the block above:

```
jest.mock('../../../../utils/dashboards-info-utils', () => ({
  getDashboardsInfo: jest.fn().mockImplementation(() => {
    return mockDashboardsInfo;
  }),
}));
```

I removed this redundant mock. Thanks for pointing that out.